Yoast/YoastSEO.js
# Add Russian transition word assessment

## Summary

This PR can be summarized in the following changelog entry: Introduce transition word assessment for Russian.

## Test instructions

This PR can be tested by following these steps: 

- check out the branch

- use the browserfied example. Don't forget to run a `grunt build:js`

- Set locale to ru_RU

- Write a Russian sentence containing a one or two-part transition word (see list of transition words in the code files)

- Hit the mark transition word sentences button to check if the sentences are correctly marked.


Fixes #1431
